# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpfe/appshell/nsIAppShellService.idl
# Commit: 765d3364cca0
# Full Hash: 765d3364cca0a67c614cb4fb5dff413a8c29d834
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-07 03:53:09
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 7.2: Create and use nsOpenWindowInfo types in nsWindowWatcher logic, r=kmag
#   
#   This patch builds on top of part 7.1 by creating this object within
#   nsWindowWatcher and ContentParent to carry the relevant information through
#   provider interfaces when creating new content windows. The nsOpenWindowInfo
# ==============================================================================

diff -r 8a13355b4ac4 -r 765d3364cca0 xpfe/appshell/nsIAppShellService.idl
--- a/xpfe/appshell/nsIAppShellService.idl	Mon Apr 06 14:29:59 2020 +0000
+++ b/xpfe/appshell/nsIAppShellService.idl	Mon Apr 06 14:30:02 2020 +0000
@@ -36,19 +36,13 @@
    *                      tag in the XUL.  Set to NS_SIZETOCONTENT to force
    *                      the window to wrap to its contents.
    * @param aInitialHeight like aInitialWidth, but subtly different.
-   * @param aOpeningTab The RemoteTab that requested that this window be opened.
-   *                    Can be left null.
-   * @param aOpenerWindow The Window Proxy which requested that this window be opened.
-   *                      Can be left null.
    */
   const long SIZE_TO_CONTENT = -1;
   nsIAppWindow createTopLevelWindow(in nsIAppWindow aParent,
                                     in nsIURI aUrl,
                                     in uint32_t aChromeMask,
                                     in long aInitialWidth,
-                                    in long aInitialHeight,
-                                    in nsIRemoteTab aOpeningTab,
-                                    in mozIDOMWindowProxy aOpenerWindow);
+                                    in long aInitialHeight);
 
   /**
    * This is the constructor for creating an invisible DocShell.